charts/kubernikus-system/requirements.yaml

```diff
@@ -14,3 +14,6 @@ dependencies:
   - name: prometheus-node-exporter
     repository: https://kubernetes-charts.storage.googleapis.com/
     version: 0.3.0
+  - name: fluent-bit
```

Comment:
I would like it better if we vendor forked charts in the `charts/` subdirectory. Its more approachable to further changes and also much easier to take a look what the cart actually contains.